README.md

```diff
@@ -599,15 +599,16 @@ For more information about multi-instrumentation feature capabilities please see
 The OpenTelemetry Operator comes with an optional component, the [Target Allocator](/cmd/otel-allocator/README.md) (TA). When creating an OpenTelemetryCollector Custom Resource (CR) and setting the TA as enabled, the Operator will create a new deployment and service to serve specific `http_sd_config` directives for each Collector pod as part of that CR. It will also rewrite the Prometheus receiver configuration in the CR, so that it uses the deployed target allocator. The following example shows how to get started with the Target Allocator:
 
 ```yaml
-apiVersion: opentelemetry.io/v1alpha1
+kubectl apply -f - <<EOF
```

Comment:
Do we rly need this `kubectl apply -f - <<EOF \n EOF` stuff?

Reply:
it's used across the readme, but it was missing here

Reply:
It's nice for getting started